lighthouse-cli/test/smokehouse/source-maps/expectations.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const pathToMap = path.join(__dirname, '../../fixtures/source-maps/bundle.js.map');
+const map = JSON.parse(fs.readFileSync(pathToMap, 'utf-8'));
+
+/**
+ * Expected Lighthouse audit values for source map tests.
+ */
+module.exports = [
+  {
+    artifacts: {
+      SourceMaps: [{
+        scriptUrl: 'http://localhost:10200/source-maps/source-maps-tester.html',
+        errorMessage: 'SyntaxError: Unexpected token { in JSON at position 1',
+        map: undefined,
+      }, {
+        scriptUrl: 'http://localhost:10200/source-maps/source-maps-tester.html',
+        map,
```

Comment:
what about asserting stuff inline instead of just that it fetched this file which matches itself? I see the value in checking for the entire match, but it also means if e.g. the file gets misplaced it could happily be checking the `source-maps` is indeed fetching an empty string, or `null` or whatever

Could do a mix and match (assert a few things, defer to the object for the rest)

Reply:
added some assertion on the keys for `map`.

if the file is moved or something, this will error, either on fs.readFileSync or the key assertions